README.md

```diff
@@ -1,11 +1,62 @@
-# sinol-make
-CLI tool for creating sio2 task packages. \
-Currently in development and not yet ready to be used.
+# <img src="https://avatars.githubusercontent.com/u/2264918?s=200&v=4" height=60em> sinol-make
 
-## Installing from source
-`pip3 install .`
+`sinol-make` is a CLI tool for creating and verifying problem packages
+for [sio2](https://github.com/sio2project/oioioi)
+with features such as:
+- measuring time and memory in the same deterministic way as sio2,
+- running the solutions in parallel,
+- keeping a git-friendly report of solutions' scores,
+- catching mistakes in the problem packages as early as possible,
+- and more.
+
+# Contents
+
+- [Why?](#why)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Configuarion](#configuration)
+- [Reporting bugs and contributing code](#reporting-bugs-and-contributing-code)
+
+### Why?
+
+The purpose of the tool is to make it easier to create good problem packages
+for official competitions, which requires collaboration with other people
+and using a multitude of "good practices" recommendations.
+While there are several excellent CLI tools for creating tests and solutions,
+they lack some built-in mechanisms for verifying packages and finding mistakes
+before uploading the package to the judge system.
+As sinol-make was created specifically for the sio2 problem packages,
+by default it downloads and uses sio2's deterministic mechanism of measuring
+solutions' runtime, called `oiejq`.
```

Comment:
```suggestion
solutions' runtime, wrapped as `oiejq`.
```
Would that be closer to the truth? It might now be harder to understand the sentence though.

Reply:
Yeah, I think that it's harder to understand now. I think we should leave it as it is

Reply:
Then I'll leave it as it is. Is it ready to be merged, or do you have some more comments/suggestions?

Reply:
I think it's ready to merge